doc/rdt.md

```diff
@@ -46,12 +48,14 @@ partition may overlap (and they usually do).
 
 Requirements for class specifications:
 
-- Names of classes must be unique accross all partitions
+- Names of classes must be unique accross all partitions.
 - Total number of classes (CLOSes) supported by the underlying hardware must
   not be exceeded.
   - **NOTE:** resctrl root and possible groups managed outside goresctrl are also
   accounted against this limit.
-- Reserved name `DEFAULT` or an empty string refer to the resctrl root
+- Reserved name `DEFAULT` or an empty string refer to the resctrl root.
```

Comment:
Woops, this is outdated 😇  It's not `DEFAULT` anymore but `system/default` as you describe below

```suggestion
```